satellite/metainfo/db_test.go

```diff
@@ -35,7 +35,7 @@ func newTestBucket(name string, projectID uuid.UUID) storj.Bucket {
 		},
 		DefaultEncryptionParameters: storj.EncryptionParameters{
 			CipherSuite: storj.EncAESGCM,
-			BlockSize:   32,
+			BlockSize:   9 * 10,
```

Comment:
what was wrong with 32?

Reply:
i hit this error https://github.com/storj/storj/blob/master/pkg/metainfo/kvmetainfo/buckets.go#L81

Reply:
Do you think it might be worth using named constants for 9 and 10, or a named `storj.RedundancyScheme` structure above, so it can be more clear that the intent here is `rs.ShareSize * rs.RequiredShares`? Or just a comment maybe.